### PR TITLE
Feature/308 warning web page

### PR DESCRIPTION
**Pull request by ruilealsilva**
Screenshot 2019-10-25 at 17 47 47

### CODE REVIEW

File: frontend/src/pages/Web/index.js [hunk 1]
```diff
@@ -31,13 +28,6 @@ const WebsiteTitle = styled('h3')({
 });
 
 class Web extends Component {
-  handleWebsiteClick = websiteId => {
```
Comment (bendiggle) [inline frontend/src/pages/Web/index.js]: What's the reason for removing this? Without this function it breaks the `/web` page.
```
TypeError: this.handleWebsiteClick is not a function
```

File: frontend/src/reducers/exerciseReducer.js [hunk 2]
```diff
@@ -8,6 +8,7 @@ const INITIAL_STATE: ExerciseState = {
   threads: [],
   activeThread: '',
   inlineNotification: null,
+  url: null,
```
Comment (bendiggle) [inline frontend/src/reducers/exerciseReducer.js]: I don't think we need this

File: frontend/src/pages/Web/index.js (hunk 1 above)
Comment (ruilealsilva) [reply frontend/src/pages/Web/index.js]: There was no reason at all. That shouldn't had been removed. My bad. (maybe some sort of multi cursor or while rebasing I lost it).

File: frontend/src/pages/Web/index.js [hunk 3]
```diff
@@ -54,7 +50,4 @@ class Web extends Component {
   }
 }
 
-export default connect(
```
Comment (bendiggle) [inline frontend/src/pages/Web/index.js]: Need to add this back too

**Pull request merged**